packages/cli/src/lib/snapshot/snapshotReporter.ts

```diff
@@ -132,12 +122,10 @@ export class SnapshotReporter {
       statuses.successes.push(SnapshotReportStatus.NO_CHANGES);
     }
 
-    if (this.isSuccessReport()) {
-      statuses.successes.push(SnapshotReportStatus.SUCCESS);
-    }
-
     if (this.resourceInErrorCount > 0) {
       statuses.errors.push(SnapshotReportStatus.ERROR);
+    } else {
+      statuses.successes.push(SnapshotReportStatus.SUCCESS);
```

Comment:
This is bound to really evolve as we improve the error handling with better warnings, auto-fixes etc.